corehq/apps/es/users.py

```diff
@@ -163,17 +163,20 @@ def primary_location(location_id):
     )
 
 
-def location(location_id):
+def location(location_id, include_web_users=False):
```

Comment:
I do think that defaulting this filter to the old, bad behavior is likely to cause us to introduce this same issue in future work, since it's pretty surprising that `UserES().location()` excludes web users.

In your PR comment, you mentioned `mobile_user_ids_at_locations` deliberately uses this old behavior to exclude web users.  A much cleaner approach is to add `.mobile_users()` to that utility to preserve that behavior.

I found only 12 references to this filter (`$ ag '\.location\('`), so I don't think it'd be too hard to add that filter to those places and avoid the need to set this unfortunate default option.

Reply:
>  A much cleaner approach is to add .mobile_users() to that utility to preserve that behavior.

Good idea!